src/__tests__/xhr.js

```diff
@@ -0,0 +1,72 @@
+import { xhr } from '../send-request'
+
+const generateXmlHttpRequestMock = (status = 502) => ({
+    open: jest.fn(),
+    setRequestHeader: jest.fn(),
+    onreadystatechange: jest.fn(),
+    send: jest.fn(),
+    readyState: 4,
+    responseText: JSON.stringify('something here'),
+    status: status,
+})
+
+const generateXhrParams = (url, markRequestFailed) => ({
+    url,
+    data: '',
+    headers: {},
+    options: {},
+    captureMetrics: {
+        incr: () => {},
+        startRequest: () => {},
+        decr: () => {},
+        finishRequest: () => {},
+        markRequestFailed,
+    },
+    callback: () => {},
+    retriesPerformedSoFar: null,
+    retryQueue: {
+        enqueue: () => {},
+    },
+})
+
+describe('sending data with xhr', () => {
+    describe('when sentry is available on Window', () => {
+        let mockSentry
```

Comment:
Other tests use given2 for this sort of thing.

Rather than rely on opaque global variables that's a much saner way to approach this. Check out other tests for examples